_rules/role-required-states-and-properties-4e8ab6.md

```diff
@@ -50,9 +48,9 @@ This rule relies on browsers and assistive technologies to support leaving out [
 
 ## Background
 
-Omitting required ARIA properties is often the result of a developer error. When required properties are missing some browsers and assistive technologies will guess the property, or leave the element inaccessible.
+Omitting [WAI-ARIA required states and properties][] is often the result of a developer error. When required properties are missing and a default value is not specified by [WAI-ARIA Specifications](#wai-aria-specifications), the behavior is not defined. For [WAI-ARIA 1.2][], the only [explicit semantic roles][explicit semantic role] with a required property with a default value are the `option` and `tabs roles` for the `aria-selected` property.
```

Comment:
I do see the need for the new bit about required properties. I fail a bit to make the connection of this as a single paragraph 😅 I feel there could be an extra sentence to smooth the transition, or something like that, but I'm not sure exactly what nor have any good suggestion for it… 😕